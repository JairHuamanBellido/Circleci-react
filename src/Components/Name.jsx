import React, { Component } from 'react';

class Name extends Component {
    render() {
        return (
            <div>
                <h2>Hola, soy jair</h2>      
            </div>
        );
    }
}

export default Name;