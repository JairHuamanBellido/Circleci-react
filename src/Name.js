import React from "react";


export default class Name extends React.Component {

    render() {


        return (
            <h1>Hola {this.props.name}</h1>
        )

    }
}
