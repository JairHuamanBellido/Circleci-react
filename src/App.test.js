
import React from 'react';
import renderer from 'react-test-renderer';
import Name from "./Name";


describe('My Test Suite', () => {
  it('My Test Case', () => {
    expect(false).toEqual(true);
  });
});


describe('Name', ()=>{
  it('Contain Name', ()=>{
    const component = renderer.create(<Name name="hola"/>);
    let tree =  component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})