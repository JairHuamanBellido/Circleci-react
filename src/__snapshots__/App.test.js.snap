// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Name Contain Name 1`] = `
<h1>
  Hola 
  hola
</h1>
`;
